### ISSUE TITLE
Freeze reading very long line in Notepad++

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#13432]
### Steps to reproduce:

1. In Notepad++, go to a line that contains many characters (thousands, I was trying to edit an html file)
2. Observe that NVDA hangs / crashes. Notepad++ itself does not crash, but NVDA stops speaking.
3. Try going to the desktop and restarting NVDA, e.g finding the icon and pressing enter

Alternatively, reproduce #12125. This happened to me a few days ago, and it also caused similar symptoms.

### Actual behavior:

NVDA is unable to restart. That's my guess because there is no speech at all even after trying to press enter several times on the icon. 

I was only able to get speech back by restarting the entire machine.

Unfortunately, there are no log entries in nvda-old.log that would explain the crash, everything that I see there is well before the time the crash happened.

There is also no crash dump in the `%temp%` folder.

### Expected behavior:

NVDA should ideally not even be crashing. But failing that, it should not crash so badly that a full restart of the machine is required for recovery.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-24876,2325be89

#### Windows version:
10.0.19044.1526

#### Name and version of other software in use when reproducing the issue:
Notepad++ 8.3.2

#### Does the issue still occur after restarting your computer?
Yes, I have seen this happen multiple times

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by thgcode** [comment]
Please see issue #4669 , on there I explain what I discovered of this issue, but a workaround I found that fixed this problem temporarily is to remove the builtin.dic dictionary from NVDA's folder until the bug is not resolved.

**Comment by Neurrone** [comment]
Thanks @thgcode. What effects does removing that dictionary have?

**Comment by Brian1Gaff** [comment]
Could you enable narrator and tell what is going on that way?
 Sometimes on odd crashes like this logging out of the user and back in 
again seems to fix things.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "Dickson Tan" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Saturday, March 05, 2022 2:39 PM
Subject: [nvaccess/nvda] NVDA cannot be restarted without restarting the 
whole computer after certain crashes (Issue #13432)

**Comment by seanbudd** [comment]
Closing as a duplicate of #4669

**Comment by feerrenrut** [comment]
Since this issue has the template, is newer, and includes a real world use case, re-opening it and closing #4669.

### Notes from #4669
- Another previously overlooked use case: https://github.com/nvaccess/nvda/issues/4669#issuecomment-489464407 (firefox / chrome and onecore)
- Some investigation indicates regex in speech dictionaries, or at least perf issues in `speechDictHandler.processText` may be contributing.

**Comment by britechguy** [comment]
This same issue occurs when you open the "About" dialog in any Microsoft Office program of your choosing.  I've replicated it in Microsoft Word and Microsoft Excel 2016.  [Full Word Version:  Microsoft® Word 2016 MSO (Version 2209 Build 16.0.15629.20196) 32-bit]

See also discussion regarding this on the NVDA Group in Topic: [Call for Replication: NVDA goes silent in Microsoft Word About Word or Excel dialog](https://nvda.groups.io/g/nvda/topic/call_for_replication_nvda/94386293)

When I am using a OneCore voice if I wait approximately 45 seconds NVDA eventually starts reading.  That is, however, an eternity in practical terms and not an amount of time any typical user would even think to wait.  They'd presume (and rightly) that a crash of some sort has occurred.

All one needs to do to make this occur in Office is have any one of the suite's programs open then hit ALT + F, D, A
Afterward, you appear to be dead in the water unless you wait long enough, which I only knew because Tyler Spivey mentioned this in the previously noted NVDA Group topic.

**Comment by marrie** [comment]
I'm not having the issue under windows `11 with one core or eloquence. Here is my debug log attached in case anyoen wants to see it, in case there still is an error.
[nvda-old.log](https://github.com/nvaccess/nvda/files/9803351/nvda-old.log)

**Comment by seanbudd** [comment]
>This same issue occurs when you open the "About" dialog in any Microsoft Office program of your choosing. I've replicated it in Microsoft Word and Microsoft Excel 2016. [Full Word Version: Microsoft® Word 2016 MSO (Version 2209 Build 16.0.15629.20196) 32-bit]

This is a different application and likely an unrelated issue. Please open a new issue for this.

**Comment by britechguy** [comment]
This issue has been shifted around, multiple times, and part of that trail exists in this very topic.

I disagree that this is "likely an unrelated issue" and will not be opening a separate issue for it.  If someone feels it should be split off by application, that should be done "in house,"

**Comment by marrie** [comment]
What version of windows are you running? I’m running Windows 11 22H2 (x64) build 22621.521 and am not having the issue as of yet.  If this is happening with only windows 10 users this data point would be useful.

**Comment by Adriani90** [comment]
@Neurrone could you test with the last NVDA alpha and the newest Notepad++ version? Is this issue still occuring?

**Comment by Neurrone** [comment]
I won't be able to test this till all the add-ons that I use are compatible with 2023.1. This will take at least a few weeks.

**Comment by britechguy** [comment]
Would there be any utility in testing with Add-Ons disabled, at least initially.  That could give useful data if things work with "naked NVDA" and then did not after Add-Ons enter the picture?

**Comment by Adriani90** [comment]
@Neurrone one friendly reminder to test this with last versions of NVDA 2023.1 or NVDA last alpha and Notepad++ last version. otherwise we have to assume this issue is not occuring anymore. Speaking for myself, I cannot reproduce this issue.

If you are still having this issue, please attach a test document so we can investigate the cause further.

**Comment by Neurrone** [comment]
Using the file in #12125, I notice that when reading line 149, NVDA pauses for a couple of seconds before reading it in Notepad++, but this doesn't happen in Notepad.

I haven't gotten NVDA to crash though, so I'll go ahead and close this issue for now.
